src/applications/lgy/coe/components/CoeDocumentUpload.jsx

```diff
@@ -1,7 +1,129 @@
-import React from 'react';
+import React, { useReducer } from 'react';
+import Scroll from 'react-scroll';
 import FileInput from '@department-of-veterans-affairs/component-library/FileInput';
+import Select from '@department-of-veterans-affairs/component-library/Select';
+import TextInput from '@department-of-veterans-affairs/component-library/TextInput';
+
+import { getScrollOptions } from 'platform/utilities/ui';
+
+import { ACTIONS, DOCUMENT_TYPES, FILE_TYPES } from '../constants';
+import { getTopPosition } from '../utils';
+import { isValidFileType, isNotBlank, validateIfDirty } from '../validations';
+
+const initialState = {
+  documentType: DOCUMENT_TYPES[0],
+  documentDescription: {
+    dirty: false,
+    value: '',
+  },
+  errorMessage: null,
+  files: [],
+};
+
+const {
+  DOC_TYPE,
+  DOC_DESC,
+  FILE_UPLOAD_SUCCESS,
+  FILE_UPLOAD_FAIL,
+  FILE_UPLOAD_PENDING,
+  FORM_SUBMIT_FAIL,
+  DELETE_FILE,
+} = ACTIONS;
+
+const scrollToError = () => {
+  const errors = document.querySelectorAll('.usa-input-error');
+  if (errors.length) {
+    const errorPosition = getTopPosition(errors[0]);
+    const options = getScrollOptions({ offset: -25 });
+    Scroll.animateScroll.scrollTo(errorPosition, options);
+    errors[0].querySelectorAll('label')[1].focus();
+  }
+};
```

Comment:
I think you can use the platform [`scrollToFirstError` function](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/utilities/ui/index.js#L46-L59) instead

Reply:
omg I had no idea that existed 😂